packages/sdk/package.json

```diff
@@ -47,7 +47,7 @@
         "webpack-dev-server": "^3.11.2"
     },
     "engines": {
-        "node": ">=18.0.0"
+        "node": ">=20.0.0"
```

Comment:
We should be very careful here, since it might be a breaking change for those who is still on 18.x.x

Reply:
We once had an issue regarding this see https://github.com/chili-publish/studio-sdk/issues/338

Reply:
We can leave them as is, or simply do something similar 
```
"engines": {
  "node": "^18 || ^20"
}
```